File: BookingService/BookingService.Application/Models/Booking.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BookingService.Application.Models
{
    public class Booking
    {
        public string BookingId { get; set; }
        public DateTime CheckInDate { get; set; }
        public DateTime CheckOutDate { get; set; }
        public decimal TotalAmount { get; set; }
        public string Status { get; set; }
        public string UserId { get; set; }
        public string AccommodationId { get; set; }
    }
}

File: ireservation-client/src/app/models/booking.ts
import { StripeUser } from "./stripeuser";

export interface Booking {
    date: string;
    totalAmount: number;
    status: string;
    userId: string;
    accommodationId: string;
    type: string;
    dateFrom: string; 
    dateTo: string;  
    stripeUserDto?: StripeUser;
  }